rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala

```diff
@@ -282,6 +360,92 @@ trait NodeInfoServiceCached extends NodeInfoService  with Loggable with CachedRe
   }
 
   /**
+   * An utility method that gets data from backend for things that are
+   * node really nodes (pending or deleted).
+   */
+  private[this] def getNotAcceptedNodeDataFromBackend(status: InventoryStatus): Box[Map[NodeId, NodeInfo]] = {
```

Comment:
I didn't find a way to do so, without completely broking the other method. As it was one of the most used code in Rudder, I thought it was a little risky. Do you see how it could be done?